### PR TITLE
Add EPU50 IOC

### PR DESCRIPTION
**Pull request by xresende**
- Added container to EPU50 IOC. This container uses the same IOC docker image that all other pcaspy IOCs use.
- Stacked this container under `si-id-conv` container (should we keep this conv service?)
- Commented out Apt proxy usage

### CODE REVIEW

File: apps/si-id-epu50.bash [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+# CPU usage estimate (ps aux):
+
+/usr/local/bin/sirius-ioc-si-id-epu50.py \
+    --pv-prefix SI-10SB:ID-EPU50: \
+    --drive-io-port 5050 \
+    --drive-msg-port 5052 \
+    --beaglebone-addr 10.128.110.160 \
+    --autosave-dir /ioc-logs/autosave \
+    --request-file /ioc-logs/source/autosave_epu.req \
```
Comment (anacso17) [inline apps/si-id-epu50.bash:13]: this request-file must be ephemeral or permanent?

File: dockerfile-templates/Dockerfile.deps [hunk 2]
```diff
@@ -31,4 +31,8 @@ RUN pip-sirius install \
     pyvisa \
     pyvisa_py \
     urllib3 \
-    scipy
+    scipy \
+    pydantic \
+    pyaml \
```
Comment (anacso17) [inline dockerfile-templates/Dockerfile.deps:36]: typo: `pyyaml`

File: apps/si-id-epu50.bash [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+export PYTHONUNBUFFERED=yes
+
+# CPU usage estimate (ps aux):
+
+/usr/local/bin/sirius-ioc-si-id-epu50.py \
+    --pv-prefix SI-10SB:ID-EPU50: \
+    --drive-io-port 5050 \
+    --drive-msg-port 5052 \
+    --beaglebone-addr 10.128.110.160 \
+    --autosave-dir /ioc-logs/autosave \
```
[the remark this answers is not in the document]
Comment (xresende) [reply apps/si-id-epu50.bash:12]: done

File: apps/si-id-epu50.bash (hunk 1 above)
Comment (xresende) [reply apps/si-id-epu50.bash:13]: done

File: dockerfile-templates/Dockerfile.deps (hunk 2 above)
Comment (xresende) [reply dockerfile-templates/Dockerfile.deps:36]: done

**Review by rbatistacardoso — APPROVED** [review]

**Review by RafaelLyra8 — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Pull request merged**